CHANGELOG.rst

```diff
@@ -10,6 +10,9 @@ New features:
     - Enabled by default for databases that support it (mysql and postgres) with a minimum pool size of 1, and a maximum of 5
     - Not supported by sqlite
     - Can be changed by passing the ``minsize`` and ``maxsize`` connection parameters
+- Many small performance tweaks reducing filter query generation overhead by about ~5%
+- Bulk inserts are ensured to be wrapped in a transaction for >50% speedup
+- PostgreSQL prepared queries now use a LRU cache for significant >2x speedup on inserts/updates/deletes
```

Comment:
is this a significant speedup over mysql?

Reply:
I tried to do a similar thing with MySQL, but the PyMySQL driver doesn't actually support parametrized queries natively, as it re-escapes everything and submits a raw SQL for each query...

At least it encodes everything as binary, so should be injection immune, but it just seems non-ideal.

Currently PyMySQL is the only driver supporting asyncio via `aiomysql`.
Right now I'm going to keep this here, and after we do version 1.0, look at seeing if we can speed up the driver or do an asyncio wrapper around `mysqlclient` which is apparently notably faster.

Reply:
that "should be injection imune" doesn't sound very assuring, i'll let loose some injection patters on it later just in case to make sure

it's already pretty fast but any speedups to mysql would be much apreciated as that's what i personally use the most :P let me know if i can help with that bit

Reply:
That would be appreciated.